tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ActivityController.java

```diff
@@ -267,7 +268,8 @@ public Response acceptWork(
 
     @PUT
     @Path("/supports")
-    @LRA(LRA.Type.SUPPORTS)
+    @LRA(value = LRA.Type.SUPPORTS, terminal = false)
+    // TODO Verify, this seems not used by any of the TCK tests
```

Comment:
my understanding is that `LRA.Type.SUPPORTS` should not end LRA by any means

Reply:
[The method is invoked here](https://github.com/eclipse/microprofile-lra/blob/5fdeb4c35a44e441395bde8feaf69b3fd459226e/tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ActivityController.java#L298)

Reply:
@xstefank the spec says the following about `LRA.Type.SUPPORTS`

>   *  If called inside a LRA context the managed bean method execution
>   *  must then continue inside this LRA context.

ie it does not say that the implementation is not allowed to end the LRA if one was present. In other words I do not see any problem with setting `terminal = true`.